### DISCUSSION TITLE
Форми документів

### DISCUSSION THREAD

**Discussion by MusienkoAnton** [leon-mbs/zstore#670] [Q&A]
Як можно додати більш красиві форми Актів, рахунків і т.п.
Я розумію що розмітка дуже швидко формується і пристойно виводиться на термопринтер.
А от на А4, хотілось би побачити і шапку і реквізити і додаткові поля, структуру рознести.

**Comment by leon-mbs** [comment]
не знаю нашо  там  красота  але  форми  можна  кастомiзувати
там  звичайний  HTML папка  \templates\printforms\
